### ISSUE TITLE
[Feature Request] Attributes on the tag

### ISSUE THREAD

**Issue by CalebKester** [jGleitz/markdown-it-kbd#1]
I'm trying to fork your code to add attributes to the kbd tag (classes or data-attr). However I'm very green in my markdown-it plugin writing and am having trouble. I'm wondering if you know of how I could do like

``[[x CLASSNAME]]``

which would render

```<kbd class="CLASSNAME">x</kbd>```

I'd potentially want to allow multiple attributes but I figure this is an easy example to start.

**Comment by jGleitz** [comment]
I’ve just published v1.1.0 which makes this plugin compatible to [`markdown-it-attrs`](https://github.com/arve0/markdown-it-attrs).

For example, the following code:

```js
const md = require('markdown-it')();
const kbd = require('markdown-it-kbd');
const attrs = require('markdown-it-attrs');

md.use(kbd);
md.use(attrs);

console.log(md.render('[[x]]{.classname}'));
console.log(md.render('[[x]]{data-custom=test}'));
```

prints:

```html
<p><kbd class="classname">x</kbd></p>

<p><kbd data-custom="test">x</kbd></p>
```

Does this suit your needs?

**Comment by CalebKester** [comment]
Worked lovely! Thanks!
